src/steps/2.addModules/index.ts

```diff
@@ -62,13 +62,167 @@ export default defineNuxtConfig(${inspect(nuxtConfig, { compact: false })})
   // 6. Write index.vue with a nice welcome message as well as links to sub-pages
   const moduleIndexHtmlSnippets = selectedModules.map((module) => moduleConfigs[module].htmlForIndexVue).filter(html => typeof html !== "undefined")
   const moduleIndexHtml = moduleIndexHtmlSnippets.length > 0 ? "\n    " + moduleIndexHtmlSnippets.join("\n    ") : ""
+
+  const moduleIndexCSSSnippets = selectedModules.map((module) => moduleConfigs[module].cssForIndexVue).filter(html => typeof html !== "undefined")
+  const moduleIndexCSS = moduleIndexHtmlSnippets.length > 0 ? "\n    " + moduleIndexCSSSnippets.join("\n    ") : ""
+
+
   const nuxtPagesIndexVue = `<template>
```

Comment:
wild! can we put this into a seperate file to keep this one smaller?

to still be able to have this as a termplate you can also make it a functiion like so:
```
const makeNuxtPagesIndexVue = (moduleIndexHtml, moduleIndexCSS) => `suuuuper long string ${moduleIndexHtml} ....`
(
```